src/LaraguardServiceProvider.php

```diff
@@ -45,6 +44,14 @@ public function boot(Repository $config, Router $router, Dispatcher $dispatcher)
             $this->publishes([
                 __DIR__ . '/../resources/views' => resource_path('views/vendor/laraguard'),
             ], 'views');
+
+            if (! class_exists('CreateTwoFactorAuthenticationsTable')) {
+                $timestamp = date('Y_m_d_His', time());
```

Comment:
Just for the sake of Carbon, it should use `now()->format('Y_m_d_His');`

Reply:
Agreed